src/components/ThemeSwitcher/context/SwitchThemeContext.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react-refresh/only-export-components */
+
+import React, {
+  createContext,
+  useState,
+  useEffect,
+  type ReactNode,
+} from "react";
+
+interface ThemeContextProps {
+  theme: string;
+  toggleTheme?: () => void;
```

Comment:
```suggestion
  toggleTheme: () => void;
```

Ты же всегда передаешь функцию, поэтому `?` тут лишний